tests/v2/validation/prime/prime_test.go

```diff
@@ -77,15 +78,9 @@ func (t *PrimeTestSuite) TestSystemDefaultRegistry() {
 }
 
 func (t *PrimeTestSuite) TestLocalClusterRancherImages() {
-	adminClient, err := rancher.NewClient(t.client.RancherConfig.AdminToken, t.client.Session)
-	require.NoError(t.T(), err)
-
-	clusterID, err := clusters.GetClusterIDByName(adminClient, localCluster)
-	require.NoError(t.T(), err)
-
-	imageResults, imageErrors := prime.CheckLocalClusterRancherImages(t.client, t.isPrime, t.rancherVersion, t.primeRegistry, clusterID)
-	assert.NotEmpty(t.T(), imageResults)
-	assert.Empty(t.T(), imageErrors)
+	podResults, podErrors := pods.StatusPods(t.client, t.clusterName)
```

Comment:
isn't the clusterName always going to be 'local' here?

Reply:
it is, I can put it back. I just removed to more so match what we have in the provisioning refactor.